README.md

```diff
@@ -101,5 +101,252 @@ With this, we are defining a mapping from namespaces to directories. The `src/`
+    }
+
+    public function getAuthor() {
+      return $this->author;
+    }
+
+    public function getYear() {
+      return $this->year;
+    }
+}
+```
+
+To be able to use both classes, you can either import them individually:
+```
+require '../src/Model/User.php';
+require '../src/Model/Book.php';
+
+use SallePW\Model\User;
+use SallePW\Model\Book;
+
+$user = new User(1, "Nicole Marie Jimenez");
+$book = new Book("Design Systems", "Design", "Alla Kholmatova", 2017);
+
+echo $user->getName();
+echo $book->getName();
+```
+
+There is a way to simplify this. Both of these classes are in the same namespace. You have previously defined in the `composer.json` the namespaces. Instead of having to use `require` for each file individually, you can take advantage of the namespaces. First, remove the following lines:
+```
+require '../src/Model/User.php';
+require '../src/Model/Book.php';
+```
+
+Check the browser again. What error do you see? Why do you think PHP raised that error?
+
+Now, just below the `declare(strict_types=1);` line, add the `autoload.php` file which is inside the `vendor/` folder. This will load the namespaces defined in `composer.json`.
+
+```
+<?php
+
+declare(strict_types=1);
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use SallePW\Model\User;
+use SallePW\Model\Book;
+```
+
+With this, you will be able to use the `User` class and `Book` class to instantiate new objects. These two classes are the classes that you have created yourself. Now, it's time to use the classes that come from the packages or dependencies that you installed using **composer**.
+
+# Using packages
+
+First, we want to generate fake data with the [**Faker**](https://github.com/fzaninotto/Faker/blob/master/readme.md) package. According to the documentation, you can start using the package by creating the Faker:
+```
+use Faker\Factory; // using the Factory class included in the Faker package
+
+$faker = Factory::create();
+```
+
+Use this faker to create random user names. 
+```
+<?php
+
+declare(strict_types=1);
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use SallePW\Model\User;
+use SallePW\Model\Book;
+use Faker\Factory;
+
+$faker = Factory::create();
+
+// $user = new User(1, "Nicole Marie Jimenez");
+// $book = new Book("Design Systems", "Design", "Alla Kholmatova", 2017);
+
+// echo $user->getName();
+// echo $book->getName();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Composer Basics</title>
+    <style>
+        th,
+        td {
+            padding: 15px;
+            text-align: left;
+            border-bottom: 1px solid #ddd;
+        }
+    </style>
+</head>
+
+<body>
+    <h1>Library system</h1>
+    <?php
+    $user = new User(1, $faker->name());
+    $book = new Book("Design Systems", "Design", "Alla Kholmatova", 2017);
```

Comment:
I woud move that to the top. There is no pint on having this business logic here. Even if it is just instantiating the classes.